### ISSUE TITLE
Update Ubuntu in the `Dockerfile` from 18.04

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#3793]
Our development team has been forced to manually specify the `requirements.txt`. We need to update the version of Ubuntu in our `Dockerfile` to see if this resolves it.

From @maxpatiiuk:
```
we could try updating ubuntu used in docker and getting newer python - have to do that anyway, and it might just fix this
```

This update should be done anyway since standard support for Ubuntu 18.04 ended last month.

Version | Code name | Docs | Release | End of Standard Support | End of Life
-- | -- | -- | -- | -- | --
Ubuntu 20.04.6 LTS | Focal Fossa | Changes | March 23, 2023 | April 2025 | April 2023
Ubuntu 18.04.6 LTS | Bionic Beaver | Changes | September 17.2021 | June 2023 | April 2028

**Comment by maxxxxxdlp** [comment]
good idea (as long as you don't have any other burning fires right now)

I strongly suggest updating to the newest version for as many libs and dependencies as possible all at once so that we don't have to do it again for another year

**Comment by grantfitzsimmons** [comment]
This has been asked by Omer at UMass Amherst.

**Comment by TommyTran732** [comment]
Ubuntu 20.04 is pretty old now, I think you should consider 24.04 instead.

**Comment by TommyTran732** [comment]
Another thing - there are official Python versions on DockerHub. I wonder why we are doing `apt install python-xyz` on the base Ubuntu container instead of using something like the `python:alpine` container?

**Comment by grantfitzsimmons** [comment]
Another mention from Auburn University
